boxen/platforms/console.go

```diff
@@ -82,7 +82,7 @@ func NewScrapliConsole(
 	}
 
 	con := &ScrapliConsole{
-		pT:        scrapliPlatform,
+		pT:        "bandaid",
```

Comment:
this is a blocker actually.
We need `scpraligo.platform.GetPlatformName` to get this over the line I think

Reply:
yep copy, branch is I think ready to open pr. going to faff about w/ it a bit more this morning then will open and tag you -- the snippet of code in previous comment actually includes the new method too so should be easy like that. the only caveat here (for now?) is that the platform object contains the platform type method -- *not* the driver (hopefully that is clear)

Reply:
now that I look at this again. I wonder if we even need the new method? the "factory" here in boxen right now just has all the names of the platforms (because of course we have to know which is which and which we want to dispatch to) -- I dont see why we wouldn't do the same for the checkpoint box, otherwise we'd kinda have a chicken/egg situation here.

 I think we can either make the checkpoint part of scrapligo itself or just host it at a repo as mentioned in one of the other comments -- then the platform you pass in could be the platform name (like arista_eos or w/e), or the  url to load the platform definition.

Reply:
mm, trying to understand this better. Right now `pT` in the struct literal mentioned in this comment is a `string` type.
So I can't pass a slice of bytes which is a product of reading scrapli platform definition YAML.

If I keep checkpoint yaml in a gist/repo I don't see how this will help the situtation... Either `pT` needs to be an `interface{}` or checkpoint needs to be integrated in the scrapligo. But integrating checkpoint into scrapligo under a well known platform type name, such as `checkpoint_cloudguard` defeats the original purpose of having freedom where the scrapli definition yaml is hosted

Reply:
in [85dfaca](https://github.com/carlmontanari/boxen/pull/33/commits/85dfacaeec6e2a310b7838dc8f28a153b323e82a) I used a string that points to the gist defining cloudguard platform. Works just fine

**but**, this still needs to be refactored to use the platform name instead of a URL string, since scraplicfg relies on that platform name to load the appropriate cfg driver

Reply:
connected in chat, think we have a path forward here!